includes/classes/Indexable/Post/SyncManager.php

```diff
@@ -94,6 +96,134 @@ public function action_queue_meta_sync( $meta_id, $object_id, $meta_key, $meta_v
 		}
 	}
 
+	/**
+	 * When a term is updated, re-index all posts attached to that term
+	 *
+	 * @param  int       $term_id Term id.
+	 * @param  int       $tt_id Term Taxonomy id.
+	 * @param  string    $taxonomy Taxonomy name.
+	 * @since  3.5
+	 */
+	public function action_edited_term( $term_id, $tt_id, $taxonomy ) {
```

Comment:
```suggestion
	public function action__edited_term( $term_id, $tt_id, $taxonomy ) {
```

After further thoughts, I am unsure about the naming convention on these callbacks.

Their names currently say out loud which action/filter they are bound to, but there is no information about what the function actually does.

What do we think of doing something like `action__edited_term__queue_posts_for_indexing` or something similar?

Reply:
I don't disagree, but this code is going into our EP fork and should follow the existing conventions of that plugin, especially as we'd like to contribute it back upstream at some point.

Reply:
Good point. 👍